docs/fundamentals/code-analysis/quality-rules/ca1510.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "CA1510: Use ArgumentNullException throw helper"
+description: "Learn about code analysis rule CA1510: Use ArgumentNullException throw helper"
+ms.date: 11/16/2023
+f1_keywords:
+- CA1510
+helpviewer_keywords:
+- CA1510
+---
+# CA1510: Use ArgumentNullException throw helper
+
+| Property                            | Value                                          |
+|-------------------------------------|------------------------------------------------|
+| **Rule ID**                         | CA1510                                         |
+| **Title**                           | Use ArgumentNullException throw helper         |
+| **Category**                        | [Maintainability](maintainability-warnings.md) |
+| **Fix is breaking or non-breaking** | Non-Breaking                                   |
+| **Enabled by default in .NET 8**    | No                                             |
```

Comment:
What does "Enabled" here mean? Its default severity is Info / Suggestion. Does Enabled only mean Warning or Error?

Reply:
Yes, "enabled" means warning/error, probably because those are the rules that can break your build. I agree it's not entirely clear. I'll open an issue to improve it.